src/components/Logo.js

```diff
@@ -5,6 +5,7 @@ const useStyles = makeStyles(theme => ({
   logo: {
     userSelect: "none",
     position: "absolute",
+    height: 50,
```

Comment:
Yes, set the height is a good way to solve the problem, but I am thinking mayge the css property: user-event (I forgot the name) would be another approach, basically, with this property, the whole layer/div become un-interactable for the user.   (pointer-events ?)

Reply:
@mmayeda also I think if we remove the user interaction on the logo (for the mobile device), than we need to re-think the logo link, right?

Reply:
Yeah, so we can set pointerEvent: "none", however as you mentioned this would take away user interaction from the logo but it would fix the zoom controls issue if the logo interaction isn't important.

On a side note, setting height to 50 messes with the positioning on a large screen device, however if the height is closer to the actual logo container's height that won't be an issue. So 30 is probably a more appropriate height if we do choose to go down that route for this fix.

Reply:
I think for mobile device, we can remove the logo action? so using the material-ui breaking-point CSS property, we could set the pointerEvent on the mobile device?

Reply:
@mmayeda what do you think, I thought on the mobile device, it's not so necessary to click the logo, considering the screen is small.

Reply:
I think we can set the pointer events to "none" on the parent div and add pointer events "auto" to the link element to keep the link functionality.  But I'm totally fine with removing the link functionality on the smaller screen too if it becomes a source of frustration by the way of users accidentally clicking it while trying to pan around on the smaller screen

Reply:
@mmayeda good idea, also, I agree, let's disable the logo action on the mobile device, @cpaule1811 can you make this change?

Reply:
No problem.